src/main/java/com/spotinst/sdkjava/example/ElastigroupUsageExampleAzure.java

```diff
@@ -30,15 +31,15 @@ public static void main(String[] args) throws IOException {
 
         // Sleep for provisioning
         System.out.println("Sleeping... waiting for provisioning 7 seconds.");
-        sleep(7);
+        sleep(60);
         // Update group
         updateElastigroup(elastigroupClient, elastigroupId);
 
-        ElastigroupAzure group =  getGroup(elastigroupClient,elastigroupId);
+        ElastigroupAzure group =  getGroup(elastigroupClient, elastigroupId);
         String groupName = group.getName();
         String preFormat     = "groupId: %s - groupName: %s";
         System.out.println(String.format(preFormat, elastigroupId, groupName));
-
+//
         // Sleep for provisioning
         System.out.println("Sleeping... waiting for provisioning 7 seconds.");
         sleep(7);
```

Comment:
Do we need to increase the sleep time here also ?

Reply:
not required